routes/teams.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable prefer-destructuring */
+/* eslint-disable consistent-return */
+/* eslint-disable no-console */
+const express = require('express');
+const bodyParser = require('body-parser');
+const mysql = require('mysql');
+
+const router = express.Router();
+
+router.use(bodyParser.urlencoded({ extended: true }));
+router.use(bodyParser.json());
+
+const connection = mysql.createConnection({
+  host: '35.239.17.119',
+  user: 'root',
+  password: 'csc308sports',
+  database: 'clubsports',
+  multipleStatements: true,
+});
+
+connection.connect((err) => {
+  if (err) {
+    return err;
+  }
+  console.log('Connected to Database!');
+});
+
+router.get('/', (req, res) => {
+  // GET method that returns a team based on name and gender.
+  const name = req.body.name;
+  const gender = req.body.gender;
+
+  const sqlQuery = `SELECT * FROM team WHERE name = "${name}" AND gender = "${gender}";`;
+  connection.query(sqlQuery, (err, rows) => {
+    if (err) {
+      console.log(err);
+      res.sendStatus(500);
+    }
+    res.send(rows);
+  });
+});
+
+router.post('/add', (req, res) => {
+  // POST method that adds an entry into the team table.
+  const name = req.body.name;
+  const gender = req.body.gender;
+  const email = req.body.email;
+  const facebook = req.body.facebook;
+  const instagram = req.body.instagram;
+  const website = req.body.website;
+  const about = req.body.about;
+
+  const sqlQuery = `INSERT INTO team (name, gender, email, instagram, facebook, website, about) 
```

Comment:
To simplify you could write 'INSERT INTO team SET ?' and pass in all the values rather than doing them one by one

Reply:
Noted.